bin/helpers/capabilityHelper.js

```diff
@@ -241,6 +241,14 @@ const validate = (bsConfig, args) => {
       logger.warn(Constants.validationMessages.SPEC_TIMEOUT_NOT_PASSED_ERROR);
     }
 
+    if(!Utils.isUndefined(bsConfig.run_settings["record"])) {
```

Comment:
if `bsConfig.run_settings["record"]` is false then I believe, it will go inside this code block and give warnings to users if the project Id or record key is missing which it ideally shouldn't